ArduPlane/mode_autoland.cpp

```diff
@@ -152,14 +167,41 @@ bool ModeAutoLand::_enter()
 void ModeAutoLand::update()
 {
     plane.calc_nav_roll();
+
+    // check if initial climb before turn altitude has been reached
+    const float altitude = plane.relative_ground_altitude(false,false);
```

Comment:
why is this an above home alt? why not use the terrain alt if available?

Reply:
now corrected